transaction-view/src/instructions_meta.rs

```diff
@@ -81,6 +81,7 @@ pub struct InstructionsIterator<'a> {
 impl<'a> Iterator for InstructionsIterator<'a> {
     type Item = SVMInstruction<'a>;
 
+    #[inline]
```

Comment:
is there a tradeoff to inlining a fairly large function like this?

Reply:
probably some compile time cost, binary size - but for 20% performance increase (most of it came from this inlining on the "challenging" tx benchmarks) it seems almost certainly worth it to me.

This also doesn't require inlining, just suggests it and lets code outside this crate inline it as well.